dato/js/recview.js

```diff
@@ -0,0 +1,158 @@
+$(document).ready( function(event){
```

Comment:
Is the existence of jQuery guaranteed by GitBook? Will this continue to be true if we upgrade GitBook? Relying on jQuery without an explicit dependency here seems slightly fragile but I'm not sure there is a good solution either.